packages/flutter_tools/test/general.shard/test/web_test_compiler_test.dart

```diff
@@ -0,0 +1,90 @@
+import 'package:file/memory.dart';
```

Comment:
nit: license header

Reply:
Somehow I manage to forget this every single time I make a file.